app/common/notifications/notify.service.js

```diff
@@ -14,7 +14,8 @@ function Notify(_, $q, $rootScope, $translate, SliderService, ModalService) {
         confirm: confirm,
         confirmModal: confirmModal,
         confirmDelete: confirmDelete,
-        limit: limit
+        limit: limit,
+        importComplete: importComplete
```

Comment:
Hmmm, I can create a service that draws from notify.service.js but should it live next to the importer in settings? since it would be import.notify.service.js, we could later extend notify for specific elements that require special notifications.